3.figures/main_figure_4/scripts/1.find_sc_crops_top_feat.py

```diff
@@ -191,7 +191,7 @@ def generate_sc_crops(
 
 feat_import_df = pd.read_parquet(
     pathlib.Path(
-        "../../2.evaluate_models/classify_genotypes/model_evaluation_data/feature_importances.parquet"
+        "../../2.evaluate_models/model_evaluation_data/feature_importances.parquet"
```

Comment:
Is there supposed to be a feature importances figure in this pr?

Reply:
So I decided to update the software garden by moving the sc_crops folder back into the main_figure_4 folder. To check to see if that worked, I reran that notebook and had to update the path. 

This is unrelated to main figure 2, just an addition I noticed should be fixed anyway. Thanks for asking!